conda_build/windows.py

```diff
@@ -154,6 +151,16 @@ def build_vcvarsall_cmd(cmd, arch=arch_selector):
         msvc_env_lines.append('if not "%WindowsSDKDir%" == "{}" ( {} )'.format(
             WIN_SDK_71_PATH, build_vcvarsall_cmd(vcvarsall_vs_path)))
     elif version == '9.0':
+        # Get the Visual Studio 2008 path (not the Visual C++ for Python path)
+        # and get the 'vcvars64.bat' from inside the bin (in the directory above
+        # that returned by distutils_find_vcvarsall)
+        try:
+            VCVARS64_VS9_BAT_PATH = os.path.join(os.path.dirname(distutils_find_vcvarsall(9)),
+                                                'bin', 'vcvars64.bat')
```

Comment:
I'm confused. It looks like @patricksnape said this won't work in PR ( https://github.com/conda/conda-build/pull/1189 ) and had fixed it. Yet I'm seeing this showing up in [`master`]( https://github.com/conda/conda-build/blob/327d9a358e2b849a05e1cd33bb5f917dd96b9bb8/conda_build/windows.py#L169-L170 ) anyways. This is likely the cause of the issues [seen]( https://github.com/conda-forge/conda-smithy/pull/392#issuecomment-269534107 ) by @mwcraig . Was this intentional or just a bad merge?